docs/prow/prow-jobs.md

```diff
@@ -86,4 +86,17 @@ For details on how to create jobs, see:
 - [Create component jobs](./component-jobs.md)
 - [Create release jobs](./release-jobs.md)
 
-For further reference, read a more technical insight into the Kubernetes [ProwJobs](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md).
\ No newline at end of file
+For further reference, read a more technical insight into the Kubernetes [ProwJobs](https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md).
+
+## Manual job trigger
+
+If you need to trigger a new job, because re-running the job would would not use the new code base, you can do so manually.
```

Comment:
```suggestion
Re-running the job does not always mean using the new code base. In such a case, you can trigger the job manually.
```

I have a question to this one - am I understanding it correctly - does re-running a job always mean that the new code base is not used? Or just in some cases? If it's always the case I will rephrase this sentence :)

Reply:
From the current Prow page, you can get a `kubectl` command to re-run the selected job. Which means it uses the exact same commit it was running with before. If you updated something, it would not run the new code/config.
Example is maybe a job that only runs every 4 hours. To not wait for the next execution that would be 4 hours from now, you could run it manually, to verify it works.

Reply:
I updated the suggestion :)